ci-operator/step-registry/storage/create/csi-aws-efs/cross-account/storage-create-csi-aws-efs-cross-account-commands.sh

```diff
@@ -0,0 +1,312 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1091
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+# logger function prints standard logs
+logger() {
+    local level="$1"
+    local message="$2"
+    local timestamp
+
+    # Generate a timestamp for the log entry
+    timestamp=$(date +"%Y-%m-%d %H:%M:%S")
+
+    # Print the log message with the level and timestamp
+    echo "[$timestamp] [$level] $message"
+}
+
+switch_aws_credentials() {
+  local mode="$1"
+  if [[ "$mode" == "shared" ]]; then
+    export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred_shared_account"
+    logger "INFO" "Using shared AWS account(B)."
+  else
+    export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+    logger "INFO" "Using default AWS account(A)."
+  fi
+}
+
+REGION="${REGION:-$LEASED_RESOURCE}"
+if [ -z "$REGION" ]; then
+  logger "Error" "REGION is not set and LEASED_RESOURCE is empty."
+  exit 1
+fi
+
+export AWS_DEFAULT_REGION="$REGION"
+logger "INFO" "Using AWS region: $AWS_DEFAULT_REGION"
```

Comment:
I noticed in this step, `AWS_DEFAULT_REGION`, `REGION`, `[EMPTY]` are are used in `aws` command, can we consolidate them, to use `REGION` or `AWS_DEFAULT_REGION`?

Reply:
Good catch, thanks! I'll all use the `AWS_DEFAULT_REGION`, it is enough that all the steps needs in the same region.